### PR TITLE
Fix #7695: Updated text for restore tabs page

### LINKED ISSUE #7695: Text update for the restore tabs page
### Description: 
Please update the text on this page to

Brave Closed Unexpectedly
Unfortunately, Brave crashed or did not close properly. 
Restore your tabs?
Yes | No

IMG_4283

### PR DESCRIPTION
**Pull request by adityagi02**
## Summary of Changes
We need to match other apps in iOS, which states `App quit unexpectedly.`, attached screenshot for reference. All other text is same as stated in issue.

This pull request fixes #7695

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`
- [ ] New or updated UI has been tested across:
  - [ ] Light & dark mode
  - [ ] Different size classes (iPhone, landscape, iPad)
  - [ ] Different dynamic type sizes

## Screenshots:

61074599-80a28c80-a431-11e9-8e64-b5092b1ad879

safaricrashes

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — APPROVED** [review]

**Review by iccub — CHANGES_REQUESTED** [review]
Looks good. Can you update the button to say `Yes` instead of `Okay`?
It's this one `restoreTabAffirmativeButtonTitle`

**Comment by adityagi02** [comment]
Sure!

File: Sources/BraveStrings/Resources/de.lproj/BraveShared.strings [hunk 1]
```diff
@@ -2219,7 +2219,7 @@
 "RestoredFavoritesFolderName" = "Wiederhergestellte Favoriten";
 
 /* Restore Tabs Affirmative Action */
-"RestoreTabAffirmativeButtonTitle" = "Okay";
+"RestoreTabAffirmativeButtonTitle" = "Ja";
```
Comment (iccub) [inline Sources/BraveStrings/Resources/de.lproj/BraveShared.strings]: Contents in .strings files are generated automatically. You can remove it. only the change to `BraveStrings` files is what we need.
Sorry if I was not clear enough

**Review by iccub — CHANGES_REQUESTED** [review]

File: Sources/BraveStrings/Resources/de.lproj/BraveShared.strings (hunk 1 above)
Comment (adityagi02) [reply Sources/BraveStrings/Resources/de.lproj/BraveShared.strings]: np, i did the required changes.

File: Sources/BraveStrings/Resources/zh.lproj/BraveShared.strings [hunk 2]
```diff
@@ -2219,7 +2219,7 @@
 "RestoredFavoritesFolderName" = "恢复的收藏夹";
 
 /* Restore Tabs Affirmative Action */
-"RestoreTabAffirmativeButtonTitle" = "确认";
+"RestoreTabAffirmativeButtonTitle" = "是的";
```
Comment (iccub) [inline Sources/BraveStrings/Resources/zh.lproj/BraveShared.strings]: 3 more languages .strings files left to remove. Rest looks great

**Review by iccub — CHANGES_REQUESTED** [review]

File: Sources/BraveStrings/Resources/zh.lproj/BraveShared.strings (hunk 2 above)
Comment (adityagi02) [reply Sources/BraveStrings/Resources/zh.lproj/BraveShared.strings]: Thanks for pointing out, i think i made commit faster than xcode even counted it:)

**Review by iccub — APPROVED** [review]
thank you :)

**Comment by adityagi02** [comment]
When will this pr merge😅

**Comment by iccub** [comment]
now!

**Pull request merged**